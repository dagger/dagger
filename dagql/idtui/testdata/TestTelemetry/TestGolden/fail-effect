Expected stderr:

✔ connect X.Xs
│ ✔ starting engine X.Xs
│ ✔ connecting to engine X.Xs
│ ✔ starting session X.Xs

✔ load module X.Xs
│ ✔ finding module configuration X.Xs
│ ✔ initializing module X.Xs
│ ✔ inspecting module metadata X.Xs
│ ✔ loading type definitions X.Xs

✔ parsing command line arguments X.Xs

✔ viztest: Viztest! X.Xs
✘ .failEffect: Container! X.Xs
! process "sh -c echo this is a failing effect; exit 1" did not complete successfully: exit code: 1
│ ✔ container: Container! X.Xs
│ $ .from(address: "alpine"): Container! X.Xs CACHED
│ ✘ .withExec(args: ["sh", "-c", "echo this is a failing effect; exit 1"]): Container! X.Xs
│ ┃ this is a failing effect
│ ! process "sh -c echo this is a failing effect; exit 1" did not complete successfully: exit code: 1

Error logs:

✘ .withExec(args: ["sh", "-c", "echo this is a failing effect; exit 1"]): Container! X.Xs
this is a failing effect
! process "sh -c echo this is a failing effect; exit 1" did not complete successfully: exit code: 1

Setup tracing at https://dagger.cloud/traces/setup. To hide set DAGGER_NO_NAG=1
