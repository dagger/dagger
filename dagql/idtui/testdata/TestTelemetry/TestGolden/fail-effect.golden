✔ connect X.Xs
│ ✔ starting engine X.Xs
│ ✔ connecting to engine X.Xs
│ ✔ starting session X.Xs

✔ initialize X.Xs
│ ✔ resolving module ref X.Xs
│ ✔ installing module X.Xs
│ ✔ analyzing module X.Xs

✔ prepare X.Xs

✔ viztest: Viztest! X.Xs
✘ .failEffect: Container! X.Xs
│ ✔ container: Container! X.Xs
│ $ .from(address: "alpine"): Container! X.Xs CACHED
│ ✘ .withExec(args: ["sh", "-c", "echo this is a failing effect; exit 1"]): Container! X.Xs
│ ┃ this is a failing effect
│ │ ✘ exec sh -c echo this is a failing effect; exit 1 X.Xs
│ │ ! process "sh -c echo this is a failing effect; exit 1" did not complete successfully: exit code: 1

Setup tracing at https://dagger.cloud/traces/setup. To hide: export NOTHANKS=1

Error: input: container.from.withExec.sync process "sh -c echo this is a failing effect; exit 1" did not complete successfully: exit code: 1

Stdout:
this is a failing effect
Run 'dagger call fail-effect --help' for usage.
